src/Resources/MediaResource/EditMedia.php

```diff
@@ -26,6 +26,9 @@ public function getHeaderActions(): array
                 ->label(trans('curator::views.panel.edit_save')),
             Action::make('preview')
                 ->color('gray')
+                ->visible(function () {
```

Comment:
This doesn't make sense to me. If the user can't view the record then they can't even get to the EditMedia page.